lib/gollum-lib/wiki.rb

```diff
@@ -579,6 +599,16 @@ def history_sanitizer
       @history_sanitizer ||= history_sanitization.to_sanitize
     end
 
+    def redirects
+      ::Gollum::Redirects.load(self)
```

Comment:
I'm not against it per se, but find it a bit surprising that `Gollum::Redirects` is a class that doesn't have instances. Is it conceptually right to think of Redirects as a class that takes a reference to a wiki, rather than a Wiki having a redirects property? Also, for instance, if we'd want to build in caching in the future (i.e. not reading the YAML store from the repo unless `HEAD` has changed)? Again, I don't feel strongly.

Reply:
Fair points. In response, I've re-implemented `::Gollum::Redirects` as a Hash extension Module that adds the following methods to the redirects hash: `init(wiki), stale?, load, dump`. I also followed your suggestion to only reload `@redirects` if the repo `HEAD` changed since initialization of `@redirects`. Have a look and tell me what you think!